biorbd_optim/misc/optimal_control_program.py

```diff
@@ -263,16 +263,17 @@ def __init__(
             if q_mapping is not None or q_dot_mapping is not None or tau_mapping is not None:
                 raise RuntimeError("all_generalized_mapping and a specified mapping cannot be used alongside")
             q_mapping = q_dot_mapping = tau_mapping = all_generalized_mapping
-        self.__add_to_nlp("q_mapping", q_mapping, q_mapping is None, BidirectionalMapping)
-        self.__add_to_nlp("q_dot_mapping", q_dot_mapping, q_dot_mapping is None, BidirectionalMapping)
-        self.__add_to_nlp("tau_mapping", tau_mapping, tau_mapping is None, BidirectionalMapping)
+        # TODO: find a clean solution for those
+        self.__add_to_nlp("q", q_mapping, q_mapping is None, BidirectionalMapping, mapping=True)
```

Comment:
I think using a flag for the entry is a good idea, but mapping is a bit too specific. I suggest that the value is a None by default and is therefore added as a "name" attribute, or the value is a string and is thereafter added to a dictionary

Reply:
Done! I'm not sure I did exactly what you wanted, now mapping is a string named name, with default value None, and if it is not None I just add param to the dictionnary designed by name